src/AccessibilityInsights.SharedUx/Controls/CustomControls/ScannerResultCustomListControl.xaml.cs

```diff
@@ -48,6 +51,19 @@ public string DataGridAccessibleName
 
         #endregion
 
+        #region TestHeader (Dependency Property)
+
+        public string TestHeader
```

Comment:
Not a huge fan of the name here. SectionHeader, perhaps?

Reply:
I used that name because it was conflicting with SectionHeader on AutomatedChecks

Reply:
Ignore my last comment, it was just necessary to have a different name for the property, I addressed this issue in my last commit.